nix/hydra/shell.nix

```diff
@@ -35,54 +23,10 @@ let
   ++
   pkgs.lib.optionals (pkgs.stdenv.isLinux) [ pkgs.systemd ];
 
-  buildInputs = [
-    # Build essentials
-    pkgs.git
-    pkgs.pkg-config
-    cabal
-    (pkgs.haskell-nix.tool compiler "cabal-plan" "latest")
-    pkgs.haskellPackages.hspec-discover
-    # Formatting
-    pkgs.treefmt
-    fourmolu
-    cabal-fmt
-    pkgs.nixpkgs-fmt
-    tools.hlint
-    apply-refact
-    # For validating JSON instances against a pre-defined schema
-    pkgs.check-jsonschema
-    # For generating plantuml drawings
-    pkgs.plantuml
-    # For plotting results of hydra-cluster benchmarks
-    pkgs.gnuplot
-    # For integration tests
-    inputs.cardano-node.packages.${system}.cardano-node
-    # To interact with cardano-node and integration tests
-    inputs.cardano-node.packages.${system}.cardano-cli
-    # To interact with mithril and integration tests
-    inputs.mithril.packages.${system}.mithril-client-cli
-  ];
-
-  devInputs = if withoutDevTools then [ ] else [
-    # Essential for a good IDE
-    haskell-language-server
-    # The interactive Glasgow Haskell Compiler as a Daemon
-    pkgs.haskellPackages.ghcid
-    # Generate a graph of the module dependencies in the "dot" format
```

Comment:
Must: keep documentation _why_ we depend on which tool

Otherwise it will become very hard to decide whether we still need one or the other still. `graphmod` here is a good example.

Reply:
Not sure nix is the right place to do this.

Reply:
Feel free to offer me another place, but this is where the tools we use come from and hence where we documented _why_ we need them.